source/db/ar-projects.json

```diff
@@ -5097,11 +5097,11 @@
   {
     "development_stage": "released",
     "description": "توزيعة  أندرويد حرة  بالكامل.",
-    "license_url": "http://replicant.us/about/",
+    "license_url": "https://git.replicant.us/replicant/manifest/blob/replicant-6.0-dev/README.mkdn",
```

Comment:
This isn't strictly a license URL, but I guess it's the best we're going to get? Probably about the same as the old link